tests/by-util/test_touch.rs

```diff
@@ -608,7 +608,17 @@ fn test_touch_set_date_relative_smoke() {
     // > (equivalent to ‘day’), the string ‘yesterday’ is worth one day
     // > in the past (equivalent to ‘day ago’).
     //
-    let times = ["yesterday", "tomorrow", "now"];
+    let times = [
+        "yesterday",
+        "tomorrow",
+        "now",
+        "2 seconds",
+        "second",
+        "2 years 1 week",
+        "2 days ago",
+        "2 months and 1 second",
+        "fortnight",
```

Comment:
`second` and `fortnight` are already in the other `times` array a few lines above and so are tested twice.